pkg/cloudprovider/vsphere/vsphere_test.go

```diff
@@ -216,6 +217,113 @@ func TestVSphereLoginByToken(t *testing.T) {
 	vcInstance.Conn.Logout(ctx)
 }
 
+func TestAlphaDualStackConfig(t *testing.T) {
+	var vs *VSphere
+	var (
+		username = "user"
+		password = "password"
+	)
+	var testcases = []struct {
+		testName                 string
+		conf                     string
+		enableDualStackFeature	 bool
+		expectedIsSecretProvided bool
+		expectedUsername         string
```

Comment:
expectedUsername and expectedPassword are not actually checked anywhere in the test, is that expected?

Reply:
Those two are not used in that test case, going to remove that.